src/applications/simple-forms/form-upload/containers/SubmitPage.jsx

```diff
@@ -50,15 +50,34 @@ const SubmitPage = () => {
     <FormPage
       currentLocation={3}
       pageTitle={
-        <div className="vads-u-display--flex vads-u-border-bottom--2px vads-u-justify-content--space-between vads-u-align-items--center vads-u-margin-top--0">
+        <div className="vads-u-display--flex vads-u-border-bottom--1px vads-u-justify-content--space-between vads-u-align-items--center vads-u-margin-top--0">
           <h3>Uploaded file</h3>
           <a href={`/form-upload/${formNumber}/upload`}>Change file</a>
         </div>
       }
     >
-      <div>
-        <VaIcon icon="description" size={5} srtext="icon representing a file" />
-        <b>{fileName}</b> {getFileSize(fileSize)}
+      <div className="vads-u-margin-top--2">
+        <div className="vads-u-margin-y--1 vads-u-color--gray">Your file</div>
+        <VaCard style={{ maxWidth: '50%' }}>
+          <div className="vads-u-display--flex vads-u-flex-direction--row">
+            <va-icon
+              size={4}
+              icon="see name mappings here https://design.va.gov/foundation/icons"
```

Comment:
What does this string for `icon` mean?

Reply:
Yeah, that was an error. I was using a font awesome icon here and the pre-commit hook automatically swapped it out for this generic va-icon. I've since fixed it!